Doc/Manual/Extending.html

```diff
@@ -3612,6 +3612,24 @@ <H3><a name="Extending_coding_style_guidelines">42.10.15 Coding style guidelines
 should be avoided as unlike the SWIG developers, users will never have consistent tab settings.
 </p>
 
+<p>
+A newer tool, <tt>clang-format</tt>, can be used instead of <tt>indent</tt>. <a href="https://clang.llvm.org/docs/ClangFormat.html">ClangFormat</a> must be
+installed locally via the LLVM toolchain. Below is an example of how to format the emit.cxx file if an in-source build directory is used:
+</p>
+
+<blockquote>
+<pre>
+$ cd Source
+$ make format-file INDENTFILE=Modules/emit.cxx
+</pre>
+</blockquote>
+
+<p>
+Of particular note is indentation is set to 2 spaces and a tab is used instead of 8 spaces.
+The generated C/C++ code should also follow this style as close as possible. However, tabs
+should be avoided as unlike the SWIG developers, users will never have consistent tab settings.
+</p>
```

Comment:
This duplicates a paragraph above - I think we only need to say it once not once per code formatting tool.

(I actually wonder if we should just drop the stuff about `indent` - unless it's evolved it really has a C focus and does a poor job for C++.  I've tried the `beautify-file` rule and the output needs a lot of manual fixing.)

Reply:
I'll update this documentation too. Any change here I'll try merging into a new clang-format branch (since this change was to help me get the new code formatted consistently with SWIG).